### DISCUSSION TITLE
PR count should not be 0

### DISCUSSION THREAD

**Discussion by HanCai98** [anuraghazra/github-readme-stats#1932] [Q&A] [answered]
**Describe the bug**
I've made many PRs to my company's private project, but my PR count on the stats graph is 0, so weird.

**Expected behaviour**
The PR num on the stats graphs should be 30 not 0, i.e. include the PR for my company's projects

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
https://github.com/HanCai98

**Additional context**

**Comment by rickstaa** [comment, marked as answer]
@HanCai98 We do not have access to your private repositories. To have these private stats show up, you have to deploy your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats/discussions/1770#stats-card-related-questions). Further, we currently also do not include stats from organizations. This is, however, our most requested feature request (#1) and has already been implemented in #1122. It has not been released, but you can already test it out by following [these steps](https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174).

**Reply by HanCai98** [reply]
Thanks for reply! That's really helpful ~
